optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/OptaPlannerProcessor.java

```diff
@@ -84,14 +101,22 @@ HotDeploymentWatchedFileBuildItem watchConstraintsDrl() {
         return new HotDeploymentWatchedFileBuildItem(constraintsDrl);
     }
 
+    @BuildStep
+    IndexDependencyBuildItem indexDependencyBuildItem() {
+        // Needed so Annotation information in the index
```

Comment:
Soft suggestion nitpick: active tense + more concrete by calling out a specific annotation
```suggestion
        // Add @PlanningEntity and other annotations in the Jandex index for Gizmo
```